[2024-05-05T05:13:09.824+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T05:13:09.835+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample.get_files scheduled__2024-05-04T00:00:00+00:00 [queued]>
[2024-05-05T05:13:09.841+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample.get_files scheduled__2024-05-04T00:00:00+00:00 [queued]>
[2024-05-05T05:13:09.841+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T05:13:09.864+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-05-04 00:00:00+00:00
[2024-05-05T05:13:09.875+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sample', 'get_files', 'scheduled__2024-05-04T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpfj_1bafw']
[2024-05-05T05:13:09.876+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask get_files
[2024-05-05T05:13:09.880+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=31511) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-05T05:13:09.880+0000] {standard_task_runner.py:63} INFO - Started process 31513 to run task
[2024-05-05T05:13:09.919+0000] {task_command.py:426} INFO - Running <TaskInstance: sample.get_files scheduled__2024-05-04T00:00:00+00:00 [running]> on host 5bc5768c5236
[2024-05-05T05:13:10.049+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sample' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-05-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-04T00:00:00+00:00'
[2024-05-05T05:13:10.051+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T05:13:11.404+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:15.078+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:15.304+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:15.714+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:15.938+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:16.169+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:16.556+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:16.781+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:17.152+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:17.426+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:18.743+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:18.971+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:20.376+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:20.692+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:20.992+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:21.214+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T05:13:21.328+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-05T05:13:21.328+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T05:13:21.336+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=sample, task_id=get_files, execution_date=20240504T000000, start_date=20240505T051309, end_date=20240505T051321
[2024-05-05T05:13:21.384+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T05:13:21.413+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-05T05:13:21.420+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
